### PR TITLE
Fix typo issue: update examples. 

### PR DESCRIPTION
**Pull request by ashukumaNI**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-python/blob/master/CONTRIBUTING.md).

TODO: Check the above box with an 'x' indicating you've read and followed [CONTRIBUTING.md](https://github.com/ni/niveristand-python/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Fixing typo in api

### Why should this Pull Request be merged?

To fix the typo, and correct example.

### What testing has been done?

Fixed the typo and then executed the example with calibration file too.

image

### CODE REVIEW

File: src/niveristand/legacy/NIVeriStand.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 _internal.dummy()
 warnings.warn(
     "NIVeriStand.py module is deprecated. "
-    "Use only if required functionality is not yet present in niveristand.clientapi",
+    "Ashu 123 Use only if required functionality is not yet present in niveristand.clientapi",
```
Comment (skolthay) [inline src/niveristand/legacy/NIVeriStand.py:31]: was this intentional ?

File: src/niveristand/legacy/NIVeriStand.py [hunk 2]
```diff
@@ -378,7 +378,11 @@ def ConnectToSystem(self, systemdefinition_file, deploy, timeout, calibration_fi
         options = DeployOptions()
         options.DeploySystemDefinition = System.Boolean(deploy)
         options.Timeout = timeout
-        options.CalibratonFilePath = calibration_file
+        if hasattr(options, 'CalibrationFilePath'):
```
Comment (skolthay) [inline src/niveristand/legacy/NIVeriStand.py:381]: why do we need this check ?

File: src/niveristand/legacy/NIVeriStand.py (hunk 1 above)
Comment (ashukumaNI) [reply src/niveristand/legacy/NIVeriStand.py]: I was debugging. do't knw when git added the reviewers. :D

File: src/niveristand/legacy/NIVeriStand.py (hunk 2 above)
Comment (ashukumaNI) [reply src/niveristand/legacy/NIVeriStand.py]: as a part of debugging.

**Review by skolthay — APPROVED** [review]

**Pull request merged**